main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	version = "0.0.0"
+	build   = "0"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "codacy plugin"
+	app.Usage = "codacy plugin"
+	app.Version = fmt.Sprintf("%s+%s", version, build)
+	app.Action = run
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:   "token",
+			Usage:  "token for authentication",
+			EnvVar: "PLUGIN_TOKEN,CODACY_TOKEN",
+		},
+		cli.StringFlag{
+			Name:   "pattern",
+			Usage:  "coverage file pattern",
+			Value:  "**/*.out",
+			EnvVar: "PLUGIN_PATTERN",
+		},
+		cli.StringFlag{
+			Name:   "language",
+			Usage:  "language of coverage",
+			Value:  "go",
+			EnvVar: "PLUGIN_LANGUAGE",
+		},
+		cli.StringFlag{
+			Name:   "commit.sha",
+			Usage:  "git commit sha",
+			EnvVar: "DRONE_COMMIT_SHA",
```

Comment:
Would it also be helpful to add a generic `PLUGIN_COMMIT_SHA`?

Reply:
This gets always set by drone, so IMHO this doesn't make sense.